CHANGELOG.md

```diff
@@ -151,6 +151,8 @@ A very minor change in the UX of the `cdktf get` command now generates the provi
 - chore: measure and print memory consumption of build and synth steps for examples in CI [\#2060](https://github.com/hashicorp/terraform-cdk/pull/2060)
 - chore: start running tests against TF 1.2.8 and drop running against 1.0.7 [\#2058](https://github.com/hashicorp/terraform-cdk/pull/2058)
 
+Only for developers of `terraform-cdk`: we modified how the `cdktf` cli is built. This might have an effect on the generated completion script. It is recommended to build the `cdktf-cli` package and update your completion script.
```

Comment:
That's not true – users of the CDKTF could've copied this script as well 🙈 They also don't have build the CLI – running `cdktf completion` after updating will give them the latest script.

Reply:
Just checked, and our docs even describe a way of setting up completions that requires an update (as it echoes the completion script at the time of running that command into the shell config):
https://github.com/hashicorp/terraform-cdk/blob/87170fd3e46f62f62e26037ba158cc4ef52a3fa3/website/docs/cdktf/cli-reference/commands.mdx#L26-L64

We need to update this docs page as well, as it probably contains the old script.

Reply:
Can you explain a bit more about how this PR would modify normal users behavior? They'd continue to use it with the homebrew (or npm) version of cdktf, no? I'm happy to change the text, but I just wanted to understand a bit more about how this breaks. Shell autocompltion is a bit of a mystery to me still 🙈 😅

Reply:
Hi, sorry for all the fuss. When I ran your PR I must've misled myself due to the way my completions script was set-up 😅
I just ran `cdktf completion` and it outputted a script containing `cdktf` as the binary (which indicates no change and `CHANGELOG` entry is needed as it won't affect users or developers).
When the script is generated (i.e. when running `cdktf completion`) by default it's rendering referencing the `cdktf` command (which is even visible in the docs I linked above 🤦). See this line:
https://github.com/hashicorp/terraform-cdk/blob/87170fd3e46f62f62e26037ba158cc4ef52a3fa3/website/docs/cdktf/cli-reference/commands.mdx#L46

I must've adjusted this line in my local completions script manually to:
```
' reply=($(COMP_CWORD="$((CURRENT-1))" COMP_LINE="$BUFFER" COMP_POINT="$CURSOR" /Users/ansgar/projects/hashicorp/terraform-cdk/packages/cdktf-cli/bundle/bin/cdktf --get-yargs-completions "${words[@]}"))
```
(I probably did this so that my completions support the latest flags I built locally and forgot about it)

And this path (`bundle/bin/cdktf`) has been changed by this PR, but as explained, this is only my problem and not a problem for our users 😬

So, we can actually remove that line 😅
```suggestion
```